### PR TITLE
update install instructions

### PR DESCRIPTION
**Pull request by Erin-Cecele**
Thank you for contributing to the Quire Documentation & Website! Please complete the form below to submit your pull request for review.

*For the Title of this pull request, please use the format "Type/Issue-#: Brief description." For Type, the options are Edit, Add, Translate. Issue-# is only needed if this pull request addresses an existing issue.*

### Checklist 

Please put an X within the brackets that apply `[X]`.

- [X] I have read the [CONTRIBUTING.md](https://github.com/thegetty/quire/blob/main/CONTRIBUTING.md) file.

- [X] I have made my changes in a new branch and not directly in the main branch

- [X] I am requesting feedback on a draft pull request

### Is this pull request related to an open issue? If so, what is the issue number?

#532 

### Please describe the goal of this pull request and the changes that were made.

Update instructions to the best of my abilities

### CODE REVIEW

File: content/docs-v1/install-uninstall.md [hunk 1]
```diff
@@ -7,181 +7,72 @@ aliases:
...
-## macOS Installation
-
-Installing and running Quire requires using the Terminal {{< q-def "command-line shell" >}}. Open it from your Applications/Utilities folder or by pressing Command–Space Bar and typing “Terminal”. If you are new to the command-line, read our tutorial [*Working in a Command-line Shell*](/learn/tutorial/#1-working-in-a-command-line-shell).
+There is no user interface for Quire so it may look or feel different than other publishing software. This was an intentional decision. The transparency of Quire's backend provides more flexibility and opportunities for customization.
 
-### Quick Install
+When you install Quire what you are actually installing is something called a {{< q-def "command-line interface" >}} or CLI. The CLI enables you to create, preview and output publications using simple commands like `quire new`, `quire preview` and `quire build`.
 
-If you’re eager to get started, this will install the complete Quire package but without e-book or PDF output capability. These may be added later, by following steps 3–5 in the Full Install guidelines below.
+Quire is completely free and open source. Installation instructions can be found below.
```
Comment (geealbers) [inline content/docs-v1/install-uninstall.md:14]: Should we create a glossary term for "open source" and/or link to the actual open source license?

File: content/docs-v1/install-uninstall.md [hunk 2]
```diff
@@ -7,181 +7,72 @@ aliases:
...
-    ```text
-    quire --help
-    ```
-
-    Search or post to our [Discussions Forum](https://github.com/thegetty/quire/discussions) to troubleshoot installation issues.
-
-{{< q-class "box warning" >}}
-- If you are working on the macOS Monterey, you will need to also install **Python 2**. Follow this link: [https://www.python.org/downloads/release/python-2718/](https://www.python.org/downloads/release/python-2718/), click the "macOS 64-bit installer," and follow the install prompts.
+{{< q-class "box tip" >}}
+- If you are using the legacy version of Quire (v0), you will need to install the **v14.18.1 release** of Node.js: [https://nodejs.org/download/release/v14.18.1/node-v14.18.1.pkg](https://nodejs.org/download/release/v14.18.1/node-v14.18.1.pkg). Node Version Manager (NVM) is a software that can be used to toggle back-and-forth with Node.js versions. More inforamtion can be found at [https://github.com/nvm-sh/nvm](https://github.com/nvm-sh/nvm).
```
Comment (geealbers) [inline content/docs-v1/install-uninstall.md:37]: I think something like this alternate wording might get more at what you're trying to say here?

> "If you will still be using the legacy version of Quire (v0) on past projects, you will need to continue to run those with **v14.18.1** of Node.js...."

File: content/docs-v1/install-uninstall.md [hunk 3]
```diff
@@ -7,181 +7,72 @@ aliases:
...
-3. Delete the `content`, `data`, and `static` folders in the new project, and replace them with those from your old project.
-4. Copy over any style customization changes you made in the `themes/default/layouts/source/css/variables.scss` file or others. (In older versions of Quire this file would be `themes/quire-starter-theme/layouts/source/css/variables.scss`.)
+## Uninstall Quire v1
 
-Run `quire preview --verbose` to test. If you see errors logged in the command-line, or if the preview doesn’t display as intended on http://localhost:1313/, search our [Discussions Forum](https://github.com/thegetty/quire/discussions) for help.
+In your command-line shell (Terminal on macOS and PowerShell in Administrator mode on Windows), type the following text and hit enter:
 
-## Uninstall Quire
+```text
+npm uninstall --global @thegetty/quire-cli@pre-release
```
Comment (geealbers) [inline content/docs-v1/install-uninstall.md:69]: You don't need/want the version label/number for the uninstall command. Just:

```
npm uninstall --global @thegetty/quire-cli
```

**Review by geealbers — CHANGES_REQUESTED** [review]

File: content/docs-v1/install-uninstall.md (hunk 1 above)
Comment (Erin-Cecele) [reply content/docs-v1/install-uninstall.md:14]: I like the idea of linking to the license. Where can I find it? I don't see it in the repo anywhere, and can't remember the actual CC type we are going with.

**Review by geealbers — APPROVED** [review]

File: content/docs-v1/install-uninstall.md (hunk 1 above)
Comment (geealbers) [reply content/docs-v1/install-uninstall.md]: The OS license we're using is the [3-clause BSD](https://opensource.org/licenses/BSD-3-Clause). I just submitted a PR for it to be added to the quire repo. Tomorrow, when the `main` branch is updated from `feature/11ty`, the OS license will be at https://github.com/thegetty/quire/blob/main/LICENSE.

**Pull request merged**